### ISSUE TITLE
decouple external from shelf temperature

### ISSUE THREAD

**Issue by bizzinho** [SPLIfA/snow#8]

**Comment by ltdeck** [comment]
Yes, this is indeed a good thing to integrate. It will be very useful to work with different external temperatures, maybe even make it vial specific. That would allow us to integrate some anisotropy (i.e. the side close to the door will experience a higher Text).
